src/services/pcn-iptables/src/Utils.cpp

```diff
@@ -608,7 +612,9 @@ void Chain::horusFromRulesToMap(
 
   for (auto const &rule : rules) {
     i++;
-    fromRuleToHorusKeyValue(rule, key, value);
+    if (i > HorusConst::MAX_RULE_SIZE_FOR_HORUS)
+      break;
+    if(fromRuleToHorusKeyValue(rule, key, value)) {
```

Comment:
What about refactoring a little bit this?

```
if (!fromRuleToHorusKeyValue(rule, key,value) 
  break;
```

That else and return at the end is quite difficult to read.

Reply:
Thanks for the suggestion. I agree it is more clear.